test/integration/backward_compatible/parallel/sql/DataTypeTest.js

```diff
@@ -90,6 +90,12 @@ describe('SQLDataTypeTest', function () {
         isCompactCompatible = !((await TestUtil.compareServerVersionWithRC(RC, '5.2.0')) >= 0
             && !TestUtil.isClientVersionAtLeast('5.2.0'));
 
+        // Compact serialization 5.2 server configuration changes
+        if ((await TestUtil.compareServerVersionWithRC(RC, '5.2.0')) < 0) {
```

Comment:
Can you assign `await TestUtil.compareServerVersionWithRC(RC, '5.2.0')` to a constant and reuse it?

Reply:
Done.